pkg/lib/queueinformer/queueinformer_operator.go

```diff
@@ -96,6 +96,7 @@ func (o *Operator) Run(stopc <-chan struct{}) error {
 	for _, queueInformer := range o.queueInformers {
 		go o.worker(queueInformer)
 	}
+	ready <- struct{}{}
```

Comment:
When I was looking at this, I thought it might be good to check if the ready channel was nil and if so skip sending anything. That way places that don't need to monitor the ready channel can simply pass nil and not worry about creating a ready channel that isn't going to be looked at.

Reply:
@jpeeler good idea. I was also playing around with the idea of making Run(...) non blocking and have it return ready and done channels or waitgroups.